tests/python/unittest/test_gluon_data_vision.py

```diff
@@ -229,6 +229,21 @@ def test_transformer():
     transform(mx.nd.ones((245, 480, 3), dtype='uint8')).wait_to_read()
 
 
+@with_seed()
+def test_randomtransforms():
+    from mxnet.gluon.data.vision import transforms
+
+    transform = transforms.Compose([transforms.RandomApply(transforms.Compose([transforms.Resize(300), transforms.RandomResizedCrop(224)]), 0.5)])
+    
+    img = mx.nd.ones((245, 480, 3), dtype='uint8')
```

Comment:
can you use a for loop to test different shapes and dimensions?

Reply:
It may be not necessary. The test here is to check the frequency of the composed transforms applied. Also, in the new PR, I will reduce the img dimension.